components/avatar/avatar.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- font family -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+
+
+    <!-- highlight js -->
+    <link rel="stylesheet" href="../../highlight/styles/a11y-dark.min.css">
+    <script src="../../highlight/highlight.min.js"></script>
+    <script>hljs.highlightAll();</script>
+    
+    <!--css filw -->
+    <link rel="stylesheet" href="avatar.css">
+    <title>Avatar</title>
+
+   
+</head>
+<body>
+    <div class="container">
+            <div class="heading">
+                    <h3>Avatars</h3>
+                    <p>Avatars are images that users can set as their profile picture.they're rounded and square shaped.</p>
+            </div>
+       
+        
+        <div class="profile-avatars">
+            <h3>Profile Avatar</h3>
+            <p>Add the 'avatar' class to img tag to simply create profile avatar </p>
+            <div class="outer-box">
+                <img class="avatar" src="https://picsum.photos/200/300" alt="">
+                <img class="avatar" src="https://picsum.photos/200/300" alt="">
+                <img class="avatar" src="https://picsum.photos/200/300" alt="">
+               
```

Comment:
Always give some value to **alt** attribute

Reply:
Did the changes. Please review and approve the pr